OpenRA.Mods.Common/Scripting/Properties/ConditionProperties.cs

```diff
@@ -48,10 +48,11 @@ public void RevokeCondition(int token)
 			conditionManager.RevokeCondition(Self, token);
 		}
 
-		[Desc("Check whether this actor accepts a specific external condition.")]
+		[Desc("Check whether this actor accepts a specific external condition.",
+		      "The timed parameter does nothing now and is deprecated.")]
```

Comment:
Note that i've made this change as part of #12796.  I suspect that one will be merged first, which cuts a bit of complexity out of this PR.